crates/matrix-sdk-indexeddb/src/crypto_store.rs

```diff
@@ -164,7 +166,19 @@ impl IndexeddbCryptoStore {
                 db.create_object_store(KEYS::SECRET_REQUESTS_BY_INFO)?;
 
                 db.create_object_store(KEYS::BACKUP_KEYS)?;
+            } else if old_version < 1.1 {
+                // We changed how we store inbound group sessions, the key used to
+                // be a trippled of `(room_id, sender_key, session_id)` now it's a
+                // tuple of `(room_id, session_id)`
+                //
+                // Let's just drop the whole object store.
```

Comment:
is that save? The session will just be recovered during via sync?

Reply:
Nope, we're just dropping things in the indexeddb store since it's unused as of yet. I guess I could have just made a breaking change instead.